code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -571,7 +571,7 @@
 
 /datum/chemical_reaction/corgium/on_reaction(datum/reagents/holder, datum/equilibrium/reaction, created_volume)
 	var/location = get_turf(holder.my_atom)
-	for(var/i in rand(1, created_volume) to created_volume) // More lulz.
+	for(var/i in 1 to rand(1, created_volume) to created_volume) // More lulz.
```

Comment:
```suggestion
	for(var/i in 1 to rand(1, created_volume)) // More lulz.
```

Reply:
The original code is correct tho

Reply:
indeed, but this would be cleaner/more intuitive